<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NoJodan Notes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,8">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="fr_up">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>11</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>11</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QLabel" name="lbl_tittle">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>24</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>NoJodan Notes</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="fr_down">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,1">
       <item>
        <widget class="QFrame" name="fr_notes">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Consolas&quot;;
border: 2px solid #ffffff;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTabWidget" name="tab_notes">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QTabWidget::pane { /* The tab widget frame */
    border-top: 1px solid #111010;
}
QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}
QTabBar::tab {
    background: #111010;
    border: 2px solid #C4C4C3;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 8ex;
    padding: 2px;
}
QTabBar::tab:selected {
    background: #491d88;
}
QTabBar::tab:hover{
	background: #491d88;
}
QTabBar::tab:selected {
    border-bottom-color: #111010; /* same as pane color */
}</string>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
            <property name="documentMode">
             <bool>false</bool>
            </property>
            <property name="tabsClosable">
             <bool>true</bool>
            </property>
            <property name="movable">
             <bool>true</bool>
            </property>
            <property name="tabBarAutoHide">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="fr_buttons">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Consolas&quot;;
border: 2px solid #ffffff;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QPushButton" name="btn_addNote">
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #27ae60;
	border-radius: 8px;
	border-style: none;
	color: #fff;
	font-size: 16px;
	font-weight: 600;
	letter-spacing: normal;
	line-height: 1.5;
	outline: none;
	padding: 13px 20px;
	position: relative;
	text-align: center;
	text-decoration: none;
	vertical-align: top;
	white-space: nowrap;
}

QPushButton:pressed{
  background-color: #16cc65;     
}

QPushButton:hover {
	background-color: #0b7538;
}</string>
            </property>
            <property name="text">
             <string>ADD</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_renameNote">
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #fea444;
	border-radius: 8px;
	border-style: none;
	color: #fff;
	font-size: 16px;
	font-weight: 600;
	letter-spacing: normal;
	line-height: 1.5;
	outline: none;
	padding: 13px 20px;
	position: relative;
	text-align: center;
	text-decoration: none;
	vertical-align: top;
	white-space: nowrap;
}

QPushButton:pressed {
  background-color: #f5840c;     
}

QPushButton:hover {
	background-color: #b0712e;
}</string>
            </property>
            <property name="text">
             <string>RENAME</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
